readme.md

```diff
@@ -19,6 +19,7 @@ Elements in beautiful READMEs include, but are not limited to: images, screensho
 - [choojs/choo](https://github.com/choojs/choo#readme) - Badges, clean, clear. Beautiful little menu above the fold with useful links. An FAQ inside of it for the main questions. Backers. Good language.
 - [Day8/re-frame](https://github.com/Day8/re-frame#readme) - Badges, logo, TOC, etc. Stands out by being a giant, well-written essay about the tech, how to use it, the philosophy behind it, and how it fits into the greater ecosystem.
 - [iterative/dvc](https://github.com/iterative/dvc#readme) - ReStructuredText format example, PyPi compatible. Logo, custom website-like menu. Smooth how-it-works animation. Badges. Nice contribution section with the hall-of-fame.
+- [gideonwolfe/vim.reaper](https://github.com/GideonWolfe/vim.reaper) - A Hackable, Fully Featured, Rice Friendly Neovim Configuration.
```

Comment:
Description should be sentence case rather than title case in order to be consistent with the rest of the list.

Reply:
Also, need to add `#readme` suffix to the URL, so that it points directly to the README anchor on the repo's landing page.